core/insertion_marker_manager.js

```diff
@@ -154,6 +154,8 @@ Blockly.InsertionMarkerManager = function(block, handleXY) {
   if (block.outputConnection) {
     var coord = new goog.math.Coordinate(block.outputConnection.x_, block.outputConnection.y_);
     this.handleDXY = goog.math.Coordinate.difference(handleXY, coord);
+    // Fade the dragging block to make targets easier to see.
+    this.workspace_.getBlockDragSurface().setOpacity(0.7);
```

Comment:
should this be a constant we can change?

Reply:
I think it's fine to hardcode it, we won't be changing it between editors I don't think.